src/emitters/emitter.sqns.ts

```diff
@@ -997,4 +999,19 @@ export class SqnsEmitter implements IEmitter {
     const urlParts = queueUrl.split('/')
     return urlParts[urlParts.length - 1]
   }
+
+  private saveApproximateReceiveCount(receivedMessage: Message, message: IMessage<any>) {
+    const approximateRecievedCount = this.getApproximateReceiveCount(receivedMessage);
+    if (message.messageAttributes) {
+      message.messageAttributes['ApproximateReceiveCount'] = {
```

Comment:
add this to MessageMetaData

Reply:
yes Rayyan suggested, the same. Implementing this. Thanks

Reply:
done, please check.